libsolidity/CompilerStack.h

```diff
@@ -125,7 +125,8 @@ class CompilerStack: boost::noncopyable
 		std::shared_ptr<Scanner> scanner;
 		std::shared_ptr<SourceUnit> ast;
 		std::string interface;
-		void reset() { scanner.reset(); ast.reset(); interface.clear(); }
+		bool isLibrary;
```

Comment:
I thought that `reset()` function addition was enough.
But aaaahh my C background did not let me think of a default argument in a struct :)